### PR TITLE
CSS-Grid-Classes - Frank Bishop

### PR DESCRIPTION
**Pull request by FrankBishop**
Screenshot (43)
Screenshot (42)

### CODE REVIEW

File: css-grid-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-1 {
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:10]: This should be name something like `column-third`

File: css-grid-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  width: 33%;
+}
+
+.column-2 {
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:14]: This should be name something like `column-fourth`

File: css-grid-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  width: 25%;
+}
+
+.text-1 {
+  flex-wrap: wrap;
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:19]: This can be removed

File: css-grid-classes/styles.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  flex-basis: 30%;
+}
+
+.text-2 {
+  flex-wrap: wrap;
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:24]: This can be removed

File: css-grid-classes/styles.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+.column-1 {
+  width: 33%;
+}
+
+.column-2 {
+  width: 25%;
+}
+
+.text-1 {
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:18]: This should be named something like `column-third`

File: css-grid-classes/styles.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  flex-wrap: wrap;
+  flex-basis: 30%;
+}
+
+.text-2 {
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:23]: This should be named something like `column-two-thirds`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Frank, great code so far! Please see my notes

**Comment by FrankBishop** [comment]
Hi, thanks for the feedback, I have implemented it.  Here is the updated code.
Screenshot (48)
Screenshot (49)

**Review by codingcodymiller — APPROVED** [review]
Nicely done!! 👍

**Pull request merged**